rocketc/rocketc.py

```diff
@@ -343,16 +353,30 @@ def _update_user(self, user_id, username, email):
         self._set_avatar(username)
 
     @XBlock.json_handler
-    def set_default_channel(self, data, suffix=""):
+    def create_group(self, data, suffix=""):
         """
-        This method set the default variable for the channels
+        This method allows to create a group from studio
         """
         # pylint: disable=unused-argument
-        default_channel = data["channel"]
-        if default_channel != " " and default_channel is not None:
-            default_channel = default_channel.replace(" ", "_")
-            self._create_group(default_channel)
-            self.default_channel = default_channel
+
+        group_name = data["groupName"]
+        description = data["description"]
+        topic = data["topic"]
+
+        if group_name == " " or group_name is None:
+            return
+
+        group_name = group_name.replace(" ", "_")
+        group = self._create_group(group_name)
```

Comment:
is it possible to pass the description and topic for a group when it's created? this to avoid additional requests to set description and topic

Reply:
It is not possible do that